### PR TITLE
Allow newer twisted

### PR DESCRIPTION
**Pull request by asottile**
Similar to Yelp/bravado#208

I also added some py3 compatibility

### CODE REVIEW

**Comment by prat0318** [comment]
:+1:

File: tox.ini [hunk 1]
```diff
@@ -1,39 +1,36 @@
 [tox]
-envlist = py
+envlist = py26,py27
```
Comment (analogue) [inline tox.ini:2]: Does .travis.yml need updating? https://github.com/Yelp/fido/blob/master/.travis.yml#L4

Comment (asottile) [inline tox.ini:2]: heh, not only does it need updating, it has a typo -- the environment variable is TOXENV

Comment (asottile) [inline tox.ini:2]: fixed :)

**Pull request merged**